webrtc.html

```diff
@@ -1891,10 +1891,13 @@
               </dd>
               <dt><code>setConfiguration</code></dt>
               <dd>
-                <p>The <code>setConfiguration</code> method updates the <a>ICE
-                Agent</a> process of gathering local candidates and pinging
-                remote candidates.</p>
-                <p>This call may result in a change to the state of the <a>ICE
+                <p>The <code>setConfiguration</code> method updates the
+                configuration of this <code><a>RTCPeerConnection</a></code>
+                object.  This includes changing configuration of the <a>ICE
+                Agent</a>.</p>
+                <p>Since this call can change parameters relating to gathering
+                local candidates and pinging remote candidates, it may
+                result in a change to the state of the <a>ICE
                 Agent</a>, and may result in a change to media state if it
                 results in connectivity being established.</p>
```

Comment:
According to my understanding, pooled candidates still won't be used until an ICE restart occurs. So setConfiguration by itself won't change the ICE or media state.